queries/python.scm

```diff
@@ -142,6 +142,17 @@
   (_) @value
 ) @_.domain
 
+;;!! with aaa:
+;;!       ^^^
+;;!  --------
+(with_statement
+  (with_clause
+    (with_item
+      value: (_) @value
+    )
```

Comment:
Do we want to add "name" too?

Reply:
yeah was thinking that. presumably "value" shouldn't include the `as aaa` piece?